common/holders/rootHashHolder.go

```diff
@@ -1,20 +1,33 @@
 package holders
 
+import "github.com/ElrondNetwork/elrond-go-core/core"
+
 type rootHashHolder struct {
 	rootHash []byte
+	epoch    core.OptionalUint32
 }
 
 // NewRootHashHolder creates a rootHashHolder
 func NewRootHashHolder(rootHash []byte) *rootHashHolder {
 	return &rootHashHolder{
 		rootHash: rootHash,
+		epoch:    core.OptionalUint32{},
+	}
+}
+
+// NewRootHashHolderWithEpoch creates a rootHashHolder
```

Comment:
extend the comment `by also setting the epoch of the rootHash`

Reply:
Removed this constructor (per Iulian's comment).